manual/xsite-minikube/Makefile

```diff
@@ -77,18 +77,21 @@ undeploy-operator:
 define undeploy-operator
 minikube profile ${1}
 kubectl delete deployment infinispan-operator || true
+kubectl delete role infinispan-operator || true
+kubectl delete rolebinding infinispan-operator || true
+kubectl delete serviceaccount infinispan-operator || true
```

Comment:
should we also run `kubectl delete crd infinispans.infinispan.org` ?

Reply:
and also added to the openshift makefile?

Reply:
Fixed